src/main/java/ui/services/CycleTimeService.java

```diff
@@ -22,14 +30,25 @@
 
 public class CycleTimeService extends Service<Object> {
     private Sprint sprint;
```

Comment:
the reason that was done was because the sprint object contains a number of the user stories already, so we don't have to hit the api again to get them.